frame/evm-system/src/lib.rs

```diff
@@ -147,9 +147,11 @@ impl<T: Config> Pallet<T> {
 		let is_new_account = Account::<T>::mutate_exists(who, |maybe_account| {
 			let is_new_account = maybe_account.is_none();
 
-			let account = maybe_account.get_or_insert_default();
+			let mut account = maybe_account.take().unwrap_or_default();
```

Comment:
```suggestion
			let mut account = match maybe_account {
				Some(ref mut val) => val,
				None => {
					*maybe_account = Some(Default::default());
					maybe_account.as_mut().unwrap()
				}
			};
```

How about this? Saves a copy.

Reply:
Or even better:

```suggestion
			let mut account = maybe_account.get_or_insert_with(Default::default);
```

Reply:
Thanks, the last one looks nice - https://github.com/humanode-network/frontier/pull/154/commits/8e70f8f85315ac18f96e05c365f3ac3dc6406605